microsoft-edge/toc.yml

```diff
@@ -958,47 +958,68 @@
 # /DevTools
 
 # =============================================================================
-# if top-of-page title is longer, add it to displayName comma-delimited list of lookup keywords
     - name: Microsoft Edge extensions
       items:
         - name: Overview of Microsoft Edge extensions
           href: extensions-chromium/index.md
 
+# -------------------------------------
         - name: What's new
           items:
             - name: Released features for Microsoft Edge Add-ons
               href: extensions-chromium/whats-new/released-features.md
+                  # add blogs
 
             - name: Roadmap for Microsoft Edge Add-ons
               href: extensions-chromium/whats-new/roadmap.md
 
-        - name: Getting started
+# -------------------------------------
+        - name: Support and feedback
+          items:
+            - name: Contact Microsoft Edge extensions support
+              href: extensions-chromium/publish/contact-extensions-team.md
+              displayName: feedback, log issues, enter issues, report issues, reporting issues
+
+#           if h1's, instead of h2's in the above h1:
```

Comment:
should the following 4 nodes be h1's (ie, distinct articles / .md files), or remain h2 sections in the above article?
Rendered: 
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/contact-extensions-team?branch=pr-en-us-2899

Reply:
Outdated, so entered fresh comment